contracts/consensus/Consensus.sol

```diff
@@ -301,6 +308,11 @@ contract Consensus is MasterCopyNonUpgradable, CoreStatusEnum, ConsensusI {
             "Committee formation blocksegment length must be in 256 most recent blocks."
         );
 
+        require(
```

Comment:
perhaps move up to L293 as we first check active status of core; then check if precommit is present.